great_expectations/experimental/datasources/postgres_datasource.py

```diff
@@ -403,7 +405,11 @@ def add_table_asset(
         Returns:
             The TableAsset that is added to the datasource.
         """
-        asset = TableAsset(name=name, table_name=table_name, order_by=order_by)
+        asset = TableAsset(
+            name=name,
+            table_name=table_name,
+            order_by=order_by or [],  # type: ignore[arg-type]  # empty list is allowed
```

Comment:
Why does this result in an error?

Reply:
I'm not sure. Possibly I'm using the wrong version of `mypy`, I was surprised when I saw this warning.
I'll remove the type ignore and see what happens in CI.

Reply:
```
Argument "order_by" to "TableAsset" has incompatible type "Union[List[BatchSorter], List[str], List[<nothing>]]"; expected "List[BatchSorter]"  [arg-type]
```
Nevermind that's not the issue.
The type for `order_by` is `List[BatchSorter]` but there's a custom validator on the model that can coerce a `List[str]` into a `List[BatchSorter]`.

So I'm keeping the type ignore but I'll update the note.